scavenger-frontend/src/components/snapshot/SnapshotTable.vue

```diff
@@ -158,6 +165,24 @@ export default {
           ElNotification.success({message: this.$t("message.snapshot.refresh-success")});
         })
     },
+    exportSnapshot(id) {
+      const fileName = `snapshot${id}.tsv`;
+      this.$http.get(`/customers/${this.customerId}/export/snapshot/${id}?fn=${fileName}`)
+        .then((response) => {
+          var csvFile = new Blob([response.data], {type: 'text/csv'});
+          var downloadLink = document.createElement("a");
+          downloadLink.download = fileName;
+          downloadLink.href = window.URL.createObjectURL(csvFile);
+          downloadLink.style.display = "none";
+          document.body.appendChild(downloadLink);
+          downloadLink.click();
+          window.URL.revokeObjectURL(downloadLink.href);
+          document.body.removeChild(downloadLink);
+        })
+        .catch((error) => {
+          console.log(error);
+        })
```

Comment:
I don't think the `console.log` is not enough to error inform the user.
Plz add error handling like this.
https://github.com/naver/scavenger/blob/ea85075f82f252d43482ab3181260a10f00a4eb0/scavenger-frontend/src/components/snapshot/Treemap.vue#L122-L129